packages/compass-aggregations/package.json

```diff
@@ -54,9 +54,9 @@
   },
   "devDependencies": {
     "@hot-loader/react-dom": "^16.8.4",
-    "@mongodb-js/compass-crud": "^9.0.2",
-    "@mongodb-js/compass-export-to-language": "^6.0.3",
-    "@mongodb-js/compass-field-store": "^6.0.0",
+    "@mongodb-js/compass-crud": "^9.0.4",
+    "@mongodb-js/compass-export-to-language": "^6.0.5",
+    "@mongodb-js/compass-field-store": "6.0.2",
```

Comment:
is the missing caret intentional?

Reply:
It's pulling the dependency version from `compass-indexes`: https://github.com/mongodb-js/compass/blob/master/packages/compass-indexes/package.json#L49 where the caret isn't present on the `mongodb-js/compass-field-store` version `6.0.2`. It looks like it's just always been pinned in that repo: https://github.com/mongodb-js/compass/commit/7f976e44ce5f28f46db66ca6d02c0b9247ca70c9
Going to update them all to have `^6.0.3` - nice catch!